lib/llm_composer.ex

```diff
@@ -56,13 +56,14 @@ defmodule LlmComposer do
   ## Parameters
     - `settings`: The settings for the language model, including prompts and options.
     - `msg`: The user message to be sent to the language model.
+    - `old_messages`: A list of previous messages used as a context. By default is empty.
 
   ## Returns
     - The result of the language model's response, which may include function executions if specified.
   """
-  @spec simple_chat(Settings.t(), String.t()) :: Helpers.action_result()
-  def simple_chat(%Settings{} = settings, msg) do
-    messages = get_messages(settings, msg, [], %{})
+  @spec simple_chat(Settings.t(), String.t(), messages) :: Helpers.action_result()
+  def simple_chat(%Settings{} = settings, msg, old_messages \\ []) do
```

Comment:
If you need to specify the old messages, you should use the https://hexdocs.pm/llm_composer/LlmComposer.html#run_completion/3 function instead, that accept a list of Messages (from older to newer)